supervisors/sensitivity_analysis_morris/morris.R

```diff
@@ -0,0 +1,107 @@
+require(methods)
+library("rjson")
+library("sensitivity")
+library("httr")
+
+source("supervisors/sensitivity_analysis_morris/scalarmapi.R")
```

Comment:
@Zuchens, @mpaciore - katalog skryptu R jest taki sam jak katalog executora - może przekazać tą ścieżkę przy wywołaniu skryptu (jako argurment wywołania?) Jeśli ta ścieżka nie będzie przekazana to wtedy używany będzie katalog roboczy.